src/main/java/org/springframework/samples/petclinic/rest/controller/PetRestController.java

```diff
@@ -27,6 +27,7 @@
 import org.springframework.web.bind.annotation.*;
 
 import jakarta.transaction.Transactional;
```

Comment:
One last little correction and all will be ok: remove this `import`:-D

Reply:
It's done

Reply:
Thank you